### ISSUE TITLE
How to install Docker beta?

### ISSUE THREAD

**Issue by gerroon** [zadam/trilium#496]
HI

I just did docker pull btu I think it internally downgraded Docker ( Iwas using the .31 I  eblieve) so now my syncs are broken. Can you please make a :beta release so one does not downgrade unintentionally.

**Comment by gerroon** [comment]
I think this unintentional downgrade also messed up my database I cant even sync after trying to upgrade to beta again/

**Comment by gerroon** [comment]
Yeah I get internal server error with the server too, docker pull definetely downgraded this one too.

**Comment by zadam** [comment]
Did you try to pull by the version - i.e. 0.31.2-beta?  Downgrade should not mess up the database (but it won't work) ...

**Comment by gerroon** [comment]
Well it would be nice if the tag is named :beta so that one can do regular pulls, otherwise the user need to manually track all this stuff
